tests/group/test_groups_present.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: Include create_groups_json.yml
+  ansible.builtin.import_playbook: create_groups_json.yml
+
+- name: Test groups present
+  hosts: ipaserver
+  gather_facts: false
+
+  tasks:
+  - name: Include groups.json
+    ansible.builtin.include_vars:
+      file: groups.json  # noqa 505
+
+  - name: Groups present len:{{ group_list | length }}
+    ipagroup:
+      ipaadmin_password: SomeADMINpassword
+      groups: "{{ group_list }}"
+
```

Comment:
Please insert this task:

    - name: Initialize groups_names
      ansible.builtin.set_fact:
        groups_names: []

This is needed if several of the groups tests using `groups_names` are executed in a row.